src/components/MyAccount/Settings/AccountSettingsTab.test.tsx

```diff
@@ -2,21 +2,35 @@ import { patron } from "../../../../__test__/fixtures/rawSierraAccountData"
 import AccountSettingsTab from "./AccountSettingsTab"
 import MyAccount from "../../../models/MyAccount"
 import { render, screen } from "../../../utils/testUtils"
+import * as helpers from "../../../../pages/api/account/helpers"
+import userEvent from "@testing-library/user-event"
+
+jest.spyOn(helpers, "updatePatronSettings")
 
 describe("AccountSettingsTab", () => {
+  global.fetch = jest
```

Comment:
I think this is being used for the tests in the `describe("editing"...` block so moving it there before the `it` blocks makes it clearer how/where the mocking is happening.